### PR TITLE
NO-ISSUE: Bump up the Golang version 1.20.5

### PR DESCRIPTION
**Pull request by ederign**

### CODE REVIEW

**Comment by ederign** [comment]
Please wait for @ljmotta or @tiagobento  approval

File: .github/actions/setup-env/action.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ runs:
         java-version: 11
         distribution: "zulu"
 
-    - name: "Set up GOLANG 1.19.10"
+    - name: "Set up GOLANG 1.20.5"
       uses: actions/setup-go@v3
       with:
         go-version: "1.19.10"
```
Comment (tiagobento) [inline .github/actions/setup-env/action.yml]: You need to change this one too.

**Review by tiagobento — CHANGES_REQUESTED** [review]

**Review by zdrapela — CHANGES_REQUESTED** [review]
Hi, I would consider updating the go version also in `go.mod` files, like that: https://github.com/zdrapela/kie-tools/commit/8ff287b32764e3c5f08f8e4f9848fbd29768390e

File: .github/actions/setup-env/action.yml (hunk 1 above)
Comment (ederign) [reply .github/actions/setup-env/action.yml]: my bad. fixed!

**Comment by caponetto** [comment]
⚠️ Merge after Runtimes `1.40.1.Final`

**Review by zdrapela — APPROVED** [review]

**Review by caponetto — APPROVED** [review]

**Review by ljmotta — APPROVED** [review]

**Review by yesamer — APPROVED** [review]

**Comment by caponetto** [comment]
@tiagobento can we merge this PR?

**Review by paulovmr — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]

**Pull request merged**